src/home/sorting/sorting-manager.ts

```diff
@@ -65,27 +72,26 @@ export class SortingManager {
       }
     };
 
-    // Observer to detect when children are added to the issues container
+    // Observer to detect when children are added to the issues container (only once)
     const observer = new MutationObserver(() => {
       if (issuesContainer.children.length > 0) {
         observer.disconnect(); // Stop observing once children are present
         if (searchQuery) filterIssues(); // Filter on load if search query exists
       }
     });
-
-    // Start observing the issues container for child elements
     observer.observe(issuesContainer, { childList: true });
 
-    // Add event listener for input changes to filter and update URL
+    // Debounce input to avoid frequent filtering
```

Comment:
Why do you debounce the input? I don't think this is normally a problem?

Reply:
he was trying to make cypress work haha

Reply:
I was trying to optimize some code because I thought the tests were failing because of me. I will remove the logic ASAP